source/docs/romi-robot/getting-to-know-romi.rst

```diff
@@ -0,0 +1,96 @@
+Getting to know your Romi
+=========================
+
+.. warning:: We are working hard on getting the Romi documentation complete. Please check back later.
+
+Hardware, Sensors and GPIO
+--------------------------
+
+The Romi has the following built-in hardware/peripherals:
+
+- 2x geared motors with encoders
+- 1x Inertial Measurement Unit (IMU)
+- 3x LEDs (green, yellow, red)
+- 3x pushbuttons (marked A, B and C)
+- 5x configurable GPIO channels
+
+Motors, Wheels and Encoders
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The motors used on the Romi have a 120:1 gear reduction, and a no-load speed of 150 RPM at 4.5V. The wheels have a diameter of 70mm (2.75").
```

Comment:
We should include all 4 DC motor characteristics (free speed, stall torque, stall current, and free current):

```suggestion
The motors used on the Romi have a 120:1 gear reduction, and a no-load output speed of 150 RPM at 4.5V. The free current is 0.13 amps and the stall current is 1.25 amps. Stall torque is 25 oz-in (0.1765 N-m) but the built-in safety clutch might start slipping at lower torques. The wheels have a diameter of 70mm (2.75").
```